src/components/CreateMatch/CreateMatch.js

```diff
@@ -1,40 +1,135 @@
+
+    showPlayersList = () => {
+        const database = Users;
+        return database.map((singlePLayer) => <span className="list-player" key={singlePLayer.name}>
+                <span className="list-player-options">
+                    <img className="list-player-options__option" src={captain} alt="capitan"/>
+                    <img className="list-player-options__option" src={ballPlay} alt="capitan"/>
+                </span>   
+                <span className="list-player__name">{singlePLayer.name}</span>
+                <span className="list-player__number">{singlePLayer.number}</span>
+                <span className="list-player-icons">
+                    <img className="list-player-icons__icon" src={editIcon} alt="editar jugador" onClick={this.showEditPlayer}/>
+                    <img className="list-player-icons__icon" src={deleteIcon} alt="eliminar jugador" />
+                </span>   
             </span>
-            <section className="match-main">
-                <span className="match-team">
-                    <input className="match-team__input" placeholder="Nombre de tu Equipo"></input>
-                    <input className="match-team__input" placeholder="Alias de tu  Equipo"></input>
-                    <input className="match-team__input" placeholder="Número de Jugadores"></input>
-                    <span className="match-team__line" > 4-3-2-1</span>
+        )
+    };
+
+    showEditPlayer = () => {
+        this.setState({
+            editPlayer: !this.state.editPlayer,   
+        })
+    };
+
+    render() {
+        const { name } = this.state;
+        return (
+            <div className="match">
+                <span className="match-header">
+                    <span className="match-header__title">
+                        <span>Crear Partido</span>
+                    </span>
+                    <span className="match-header__icon">
+                        <img className="match-header__icon-img" src={create} alt="crear partido" />
+                    </span>
                 </span>
-                <img className="match-main__img" src={soccerField} alt="campo de juego" />
-            </section>
-            <section className="match-configure">
-                <img className="match-configure__btn" src={players} alt="Lista jugadores" />
-                <img className="match-configure__btn" src={shirts} alt="Uniforme" />
-                <img className="match-configure__btn" src={field} alt="campo de juego" />
-                <img className="match-configure__btn" src={calendar} alt="fecha y lugar" />
-                <img className="match-configure__btn" src={configure} alt="configuracion" />
-                <img className="match-configure__btn" src={deleteTeam} alt="eliminar equipo" />
-            </section>
-        </div>
-    );
-};
+                <section className="match-main">
+                    {this.state.containerInput ? <span id="infoTeam" className="match-team">
```

Comment:
add containerInput on line 89 with that you can use containerInput instead of this.state.containerInput. Do the same with nickname and the other state values on inside render method.